src/zenml/pipelines/pipeline_decorator.py

```diff
@@ -37,6 +37,7 @@ def pipeline(
     name: Optional[str] = None,
     enable_cache: bool = True,
     requirements_file: Optional[str] = None,
+    dockerignore_file: Optional[str] = None,
```

Comment:
Wasn't this in already?

Reply:
It was in the actual decorator, but missing in the `@overload` which (at least for me in PyCharm) prevented it from showing up as an autocomplete option